h-include.js

```diff
@@ -41,35 +41,64 @@ var hinclude;
   hinclude = {
     classprefix: "include_",
 
-    show_content: function (element, req) {
-      var i, include, message, fragment = element.getAttribute('fragment') || 'body';
-      if (req.status === 200 || req.status === 304) {
-        var container = document.implementation.createHTMLDocument().documentElement;
-        container.innerHTML = req.responseText;
+    create_container: function (req) {
+      var container = document.implementation.createHTMLDocument().documentElement;
```

Comment:
What do you think about using 

``` .js
(new DOMParser).parseFromString(responseText, 'text/html')
```

It [works on IE10+](https://developer.mozilla.org/en-US/docs/Web/API/DOMParser#Browser_compatibility) and has a couple of (perhaps minor) advantages:

- properly parses the `<html>` tag (if it is present)
- custom-element detection isn't performed (only affects Chrome's custom-element implementation currently)